### ISSUE TITLE
Accessibility improvements on main site pages

### ISSUE THREAD

**Issue by zburk** [gatsbyjs/gatsby#26271]
## Summary

Running Lighthouse results in the following recommendations for the following pages:

1. https://www.gatsbyjs.org/ - Heading elements are not in a sequentially-descending order
2. https://www.gatsbyjs.org/plugins/ - Form elements do not have associated labels
3. https://www.gatsbyjs.org/features/ - Background and foreground colors do not have a sufficient contrast ratio., Heading elements are not in a sequentially-descending order
4. https://www.gatsbyjs.org/contributing/ - Heading elements are not in a sequentially-descending order

### Motivation
Users relying on screen readers will not be able to follow the page flow as well. Users with vision difficulties may have trouble reading the Features comparison table headings.

## Steps to resolve this issue

Following the recommendations from Google:
[https://web.dev/heading-order/](https://web.dev/heading-order/)
https://web.dev/label/
https://web.dev/color-contrast/

### Open a pull request

- [ ] Open a pull request with your work including the words "closes #[this issue's number]" in the pull request description

**Comment by singhshivanshu** [comment]
Does this issue exists on the latest site? i.e site released today itself?

**Comment by zburk** [comment]
Home page is resolved
Plugins page still has issue
Features page still has issues
Contributing page is resolved

**Comment by LekoArts** [comment]
Hi, thanks for the issue! The form on /plugins/ can't be fixed (the way you did in your PR) as this was inaccessible as well. Since we use react-instantsearch its an upstream issue: https://github.com/algolia/react-instantsearch/issues/2863

I'll fix the color contrast issue on the features page and it'll be deployed soon, thanks for the notice!

**Comment by MartinMa** [comment]
Is there a commit associated with the resolution of this issue?

I don't know if you'd catch this one. But on the main page, the main menu in the dropdown labeled Community, the sub text "We're so glad you want to help!" does not meet the WCAG 2.1 AA requirements for color contrast yet (see attached image).
![image](https://user-images.githubusercontent.com/123297/92087068-577a5380-edcb-11ea-8a81-c6f37eb178fe.png)
